src/models/flight_control/FGLinearActuator.cpp

```diff
@@ -0,0 +1,257 @@
+ * the terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place - Suite 330, Boston, MA  02111-1307, USA.
+ * 
+ * Further information about the GNU Lesser General Public License can also be found on
+ * the world wide web at http://www.gnu.org.
+ * 
+ * FUNCTIONAL DESCRIPTION
+ * --------------------------------------------------------------------------------
+ * 
+ * HISTORY
+ * --------------------------------------------------------------------------------
+ * 
+ * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+ * COMMENTS, REFERENCES,  and NOTES
+ * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+ * 
+ * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+ * INCLUDES
+ * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+#include <stdlib.h>
+
+#include "FGLinearActuator.h"
+#include "input_output/FGXMLElement.h"
+#include "math/FGParameterValue.h"
+
+#include "models/FGFCS.h"
+
+using namespace std;
+
+namespace JSBSim {
+    
+    /* %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+     * CLASS IMPLEMENTATION
+     * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+    
+    FGLinear_Actuator::FGLinear_Actuator(FGFCS* fcs, Element* element) : FGFCSComponent(fcs, element)
+    {
+        
+        if (element->FindElement("set")) {
+            isSetProperty = true;
+            setProperty = PropertyManager->CreatePropertyObject<double>(element->FindElementValue("set"));
+        }
+        if (element->FindElement("reset")) {
+            isResetProperty = true;
+            resetProperty = PropertyManager->CreatePropertyObject<double>(element->FindElementValue("reset"));
+        }
+        
+        ptrVersus = nullptr;
+        if (element->FindElement("versus")) {
+            string property_string = element->FindElementValue("versus");
+            ptrVersus = new FGParameterValue(property_string, PropertyManager);
+            double setVersus = ptrVersus->GetValue();
+            if (ptrVersus->IsConstant()) {
+                if (setVersus >= 0.5) {
+                    versus = 1;
+                } else if (setVersus <= -0.5) {
+                    versus = -1;
+                }
+            }
+        } else versus = 0;
```

Comment:
Initialize `ptrVersus` with a real value instead, you will save an `if` condition in the `run` method.
```suggestion
        } else ptrVersus = FGRealValue(0.0);
```